clients/src/main/java/org/apache/kafka/common/utils/Utils.java

```diff
@@ -711,6 +711,11 @@ public static void closeAll(Closeable... closeables) throws IOException {
             throw exception;
     }
 
+    public static void closeQuietly(Object obj, String name) {
```

Comment:
The loss of type safety here makes me uneasy. Maybe we should just have helper methods in the places where we need them instead of potentially exposing this hole to code that is not expecting it.